network/src/main/java/com/ztkmkoo/dss/network/http/DssHttpServerInitializer.java

```diff
@@ -0,0 +1,42 @@
+package com.ztkmkoo.dss.network.http;
+
+import com.ztkmkoo.dss.network.common.creator.DssServerHandlerCreator;
+import io.netty.channel.ChannelInitializer;
+import io.netty.channel.ChannelPipeline;
+import io.netty.channel.socket.SocketChannel;
+import io.netty.handler.codec.http.HttpRequestDecoder;
+import io.netty.handler.codec.http.HttpResponseEncoder;
+import io.netty.handler.ssl.SslContext;
+
+import java.util.Objects;
+
+class DssHttpServerInitializer extends ChannelInitializer<SocketChannel> {
+
+    private final boolean ssl;
+    private final SslContext sslContext;
+    private final DssServerHandlerCreator handlerCreator;
+
+    DssHttpServerInitializer(
+            boolean ssl,
+            SslContext sslContext,
+            DssServerHandlerCreator handlerCreator) {
+        this.ssl = ssl;
+        this.sslContext = sslContext;
```

Comment:
I wonder if i have to check sslContext is not null when ssl is true. Actually it will throw the exception when init channel.